source/Tool/GenerateAction.cpp

```diff
@@ -62,6 +64,11 @@ DoGenerateAction()
         return absPath.getError();
     auto workingDir = files::getParentDir(*absPath);
 
+    // load in plugins
+    if (auto err = loadPlugins(toolArgs.addonsDir); err.failed())
```

Comment:
you don't need the "; err.failed()" part

Reply:
I don't? Isn't how do I negate it otherwise?

Reply:
`err` evaluates to `true` if there's an error (please double check this lol I get it confused sometimes)